packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -621,7 +621,7 @@ - (UIView *)betterHitTest:(CGPoint)point withEvent:(UIEvent *)event
   //   * Taking `layer.zIndex` field into an account is not required because
   //     lists of `ShadowView`s are already sorted based on `zIndex` prop.
 
-  if (!self.userInteractionEnabled || self.hidden || self.alpha < 0.01) {
+  if (!self.userInteractionEnabled || self.hidden) {
```

Comment:
I don't think this is right... 
If `alpha == 0`, we don't want for the view to be interactive. This is also the same behavior in the native platform (opacity == 0, means that the view is not rendered and it is not interactive) and what we want to maintain.

We want interactivity if alpha is `> 0`.

Reply:
Yes, in iOS native UIView, it's the default behaviour, but we support interactivity when alpha is 0 in  iOS old arch. Also Android and Web support it. Should we need to keep consistency across platforms?

Reply:
I don't think in this case. In this case the platform should dominate.
One of the best things of React Native is that it gives a native feeling to the user. If we manipulate how the platform behaves, then people will notice when an app is built with React Native vs Native and that something we don't want.

Reply:
@cipolleschi Hmm, it seems that in iOS we can also set the background color to a clear color, which is not visible to users but responds the touch event. Therefore, for users there is not much difference between setting the alpha to 0 and setting the background color to clear. However, developers must be aware of the differences between iOS and other platforms.

Reply:
However, if it is indeed believed that iOS needs to be different. Feel free to close.